replay-events/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine as analytics-fork
+
+RUN apk --update add --no-cache git
+RUN git clone --depth=1 https://github.com/adityapahuja/matomo-log-analytics.git
+
+FROM python:2-alpine
+
+COPY --from=analytics-fork /matomo-log-analytics /log-analytics
+COPY docker-entrypoint.sh /entrypoint.sh
```

Comment:
Not completely obvious to me, what is the need for multi-stage build here?

Reply:
I think all it's doing is keeping git out of the resultant image. Probably overkill here, but originally I was doing more in other images than this. Can remove and just install git in the image if you don't think it's worth the added complexity?

Reply:
Probably not worth having a second build in there, but worth adding a one-liner to delete the `.git` directory.